manual/parts/vspace.tex

```diff
@@ -109,23 +109,19 @@ \subsubsection{AArch32}
 \subsubsection{AArch64}
 
 Depending on configuration, Arm AArch64 processors have page-table structures with 3 or 4 levels.
-The VSpace object is therefore implemented either by the \obj{PageGlobalDirectory} object (4 level
-configs) or the \obj{PageUpperDirectory} object (3 level configs). To help with writing code that is
-generic in this distinction, \texttt{libsel4} provides the macro \texttt{seL4\_ARM\_VSpaceObject}
-that expands to the correct corresponding object. All intermediate paging structures are indexed by
-9 bits of the virtual address. Depending on configuration, the top-level object is indexed by either
-9 or 10 bits. The macro \texttt{seL4\_VSpaceIndexBits} makes this value available under a generic
-name. The table below shows the four-level configuration. On three-level configurations, the
-invocations for \texttt{seL4\_ARM\_VSpaceObject} capabilities are instead available on
-\obj{PageUpperDirectory} caps.
+The VSpace object is \texttt{seL4\_ARM\_VSpaceObject}, which is a distinct object type used for the
+top level page table. All intermediate paging structures are indexed by 9 bits of the virtual
+address and are \obj{PageTable} objects. Depending on configuration, the top-level object is
+indexed by either 9 or 10 bits. The macro \texttt{seL4\_VSpaceIndexBits} makes this value available
+under a generic name. The table below shows the four-level configuration.
 
 \begin{tabularx}{\textwidth}{Xlll} \toprule
 \emph{Object}                    & \emph{Address Bits} & \emph{Level} & \emph{Methods} \\ \midrule
-    \texttt{PageGlobalDirectory}/\texttt{seL4\_ARM\_VSpaceObject}
+    \texttt{PageTable}/\texttt{seL4\_ARM\_VSpaceObject}
```

Comment:
Shouldn't this be just `seL4_ARM_VSpaceObject`? With the old code you could argue that `PageGlobalDirectory` is the same object, but that doesn't hold for `PageTable`.

Reply:
Yes, my bad.